app/models/grade.rb

```diff
@@ -37,7 +37,7 @@ def self.to_csv(assignment_id)
       attributes = %w[email name grade remarks]
       group_members = User.joins(group_members: :assignments)
                           .where(group_members: { assignments: { id: assignment_id } })
-      submissions = Project.where(assignment_id: assignment_id)&.includes(:grade, :author)
+      submissions = Project.where(assignment_id: assignment_id).includes(:grade, :author)
```

Comment:
**Suggestion** Not sure if removing & is a good idea. If this does not concern the current set of changes then better not do the same